tests/testapp/tests/test_delete_stale_comments.py

```diff
@@ -0,0 +1,30 @@
+from django.core.management import call_command
+
+from django_comments.models import Comment
+
+from . import CommentTestCase
+from testapp.models import Article
+
+
+class CommentManagerTests(CommentTestCase):
+
+    def testDoesNotRemoveWhenNoStaleComments(self):
+        self.createSomeComments()
+        initial_count = Comment.objects.count()
+
+        call_command("delete_stale_comments", "--yes", verbosity=0)
+
+        self.assertEqual(initial_count, Comment.objects.count())
+
+    def testRemovesWhenParentObjectsAreMissing(self):
+        self.createSomeComments()
+        initial_count = Comment.objects.count()
+        article_comments_count = Comment.objects.for_model(Article).count()
+        self.assertGreater(article_comments_count, 0)
+
+        # removing articles will not remove associated comments
+        Article.objects.all().delete()
+        call_command("delete_stale_comments", "--yes", verbosity=0)
+
+        self.assertEqual(0, Comment.objects.for_model(Article).count())
+        self.assertEqual(initial_count-article_comments_count, Comment.objects.count())
```

Comment:
I'm not sure I follow. I don't see any failures in CI so not sure where I may be missing spaces.

Reply:
ok, found the missing spaces around the `-` sign by running flake8 manually on this file. This needs to be in CI and if it is then the CI setup is broken because everything reported PASS.